### PR TITLE
minor copy-editing changes 2

### PR DESCRIPTION
**Pull request by orucky**
Follow-up on #38.

**Note:** only 5b48b28 (`fix typos, do some rewording`) to be reviewed!

### CODE REVIEW

File: doc/40_Configuration.md [hunk 1]
```diff
@@ -49,39 +49,42 @@ image: node:alpine
...
+initialization. To that end you can provide a shell command that gets executed in a terminal on start.
 
 For instance, the start script for the Gitpod documentation repository is defined as:
 ```yaml
 tasks:
   - command: "npm install && npm run serve"
 ```
 
-You can chain multiple shell commands with `&&`.
+Similarly to Docker, you can chain multiple shell commands with `&&`.
```
Comment (JanKoehnlein) [inline doc/40_Configuration.md]: This is not so much a Docker but a bash feature

File: doc/50_IDE.md [hunk 2]
```diff
@@ -15,11 +15,11 @@ emphasizes diversity.
 
 ## Overview
 
-When first opened, you will see a welcome screen that briefly gives you the most important pointers
-to get started. After closing it you will see the workbench shell. Depending on the
+When first open Theia, you will see a welcome screen that quickly gets you started.
```
Comment (JanKoehnlein) [inline doc/50_IDE.md]: When _you_ first open

File: doc/52_Tips_and_Tricks.md [hunk 3]
```diff
@@ -53,8 +51,8 @@ It opens the search view, which is described in more detail [here](56_Search.md)
 ## Symbols and References
 
 Most languages allow to name things and refer to them in other places, for example variables,
-functions, classes, modules etc. Gitpod supports this notion of such _symbols_ and _references_ to
-them and allows to navigate in both directions:
+functions, classes, modules etc. Gitpod supports this notion of _symbols_ (and their _references_) 
+and supports navigation in both directions:
```
Comment (JanKoehnlein) [inline doc/52_Tips_and_Tricks.md]: delete duplicate 'supports'

File: doc/54_Git.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 # Git Integration
 
-Theia ships with a couple of Git-related features. Please note that these are all meant to be
-convenient UIs on top of the command line `git`, which is available in the terminal as well.
+Theia integrates closely with Git. All of those features are convenient UIs on top of the
```
Comment (JanKoehnlein) [inline doc/54_Git.md]: Theia integrates closely with Git _by offering_ convenient UIs

File: doc/54_Git.md [hunk 5]
```diff
@@ -46,15 +46,15 @@ An unstaged change can be reverted by clicking on the `Discard Changes` action n
 location. Double-clicking on a modified file will open it in a diff editor. The read-only editor on
 the left-hand side shows the state from the index. The right-hand side of the editor reflects the
 state of the working tree, and it lets you to further modify the file. New and deleted files do not
-open in a diff editor. The files can be staged (added to the index) by clicking on the `Stage
-Changes` action (`+` sign), the opposite operation being the `Unstage Changes` action (`-` sign).
+open in a diff editor. Files can be staged (added to the index) by clicking on the `Stage
+Changes` action (`+` sign). To unstage a changes, use `Unstage Changes` action (`-` sign).
```
Comment (JanKoehnlein) [inline doc/54_Git.md]: To unstage a change...

File: doc/56_Search.md [hunk 6]
```diff
@@ -13,13 +13,13 @@ There are several ways to perform a search and to find things in the workspace:
...
-command palette or by pressing <kbd>Ctrl+Shift+F</kbd> (<kbd>⌘+Shift+F</kbd> on macOS)
+command palette or by pressing <kbd>Ctrl+Shift+F</kbd> (<kbd>⌘+Shift+F</kbd> on macOS).
 
 In the `Search` view, you can start typing, and the matching results will be shown automatically
 without pressing <kbd>Enter</kbd>. The search can be restricted to case matching, whole word
-matching, or, when desired, the query can be given as a regular expression. Search results are
+matching, or the query can be given as a regular expression. Search results are
 grouped by file, with an indication of the number of occurrences and the file's path. Expand a file
-to see a preview of all of the hits within that file. Then single-click on one of the hits to view
+to see a preview of all of the hits within that file. Then single-click on one of the files to view
```
Comment (JanKoehnlein) [inline doc/56_Search.md]: 'hits' are subnodes of the 'files', so better leave it as it was

File: doc/56_Search.md [hunk 7]
```diff
@@ -74,14 +74,14 @@ Selecting a file will open it an an editor.
 
 ## Open Workspace Symbol
 
-Press <kbd>Ctrl+O</kbd> (<kbd>⌘+O</kbd> on macOS) to find all symbols in the workspace. The meaning
-of a symbol and the available symbol kinds are language-specific.
+Press <kbd>Ctrl+O</kbd> (<kbd>⌘+O</kbd> on macOS) to find symbols in the workspace. The meaning
+of a symbol and what symbols are available is language specific.
```
Comment (JanKoehnlein) [inline doc/56_Search.md]: sure no dash in language-specific?

File: doc/58_Pull_Requests.md [hunk 8]
```diff
@@ -38,9 +36,9 @@ everything is staged, add a meaningful commit message and press the `Commit` but
 ![](images/pr-commit.png)
 
 The next step in the GitHub flow is to propose our changes as a GitHub pull request. You can do that
-right now from within Gitpod. In the `Pull Request` view you will see a message that our current
-branch has no remote tracking branch. By clicking on the `Publish Changes` button, this remote
-branch is created on GitHub and synchronized with your local branch.
+ from within Gitpod. In the `Pull Request` view you will see a message that our current
+branch has no remote tracking branch. By clicking on the `Publish Changes` button, Gitpod creates a remote
+branch GitHub and synchronizes it with our local branch.
```
Comment (JanKoehnlein) [inline doc/58_Pull_Requests.md]: branch _on_ Github...

File: doc/60_Dashboard.md [hunk 9]
```diff
@@ -4,7 +4,7 @@ Gitpod's dashboard allows you to log in and out, navigate through all your works
 details of your account and subscription. It can be reached by pointing your browser to
 [https://gitpod.io](https://gitpod.io).
 
-If you're not logged in, you will see a _Log in_ button. Otherwise, the top-right corner will
+If you are not logged in, you will see a _Log in_ button. Otherwise, the top-right corner will
```
Comment (JanKoehnlein) [inline doc/60_Dashboard.md]: Did you do a find/replace for all occurrences of _don't_, _can't_, _doesn't_, _isn't_, _it's_ etc?

File: doc/50_IDE.md (hunk 2 above)
Comment (orucky) [reply doc/50_IDE.md]: fixed that independently in the follow-up

File: doc/40_Configuration.md (hunk 1 above)
Comment (orucky) [reply doc/40_Configuration.md]: Was thinking the same. But concluded we probably want to point out to the analogy ppl will most likely know about ...

File: doc/52_Tips_and_Tricks.md (hunk 3 above)
Comment (orucky) [reply doc/52_Tips_and_Tricks.md]: also fixed in the follow-up

File: doc/54_Git.md (hunk 4 above)
Comment (orucky) [reply doc/54_Git.md]: good tip, I fill fix my fix in the follow-up

File: doc/54_Git.md (hunk 5 above)
Comment (orucky) [reply doc/54_Git.md]: used *unstage changes* in the follow-up. Is that OK?

File: doc/40_Configuration.md (hunk 1 above)
Comment (orucky) [reply doc/40_Configuration.md]: @32leaves ?

File: doc/56_Search.md (hunk 6 above)
Comment (orucky) [reply doc/56_Search.md]: OK, fixed in the follow-up

File: doc/56_Search.md (hunk 7 above)
Comment (orucky) [reply doc/56_Search.md]: it should be there, adding to the follow-up

File: doc/58_Pull_Requests.md (hunk 8 above)
Comment (orucky) [reply doc/58_Pull_Requests.md]: fixed

File: doc/60_Dashboard.md (hunk 9 above)
Comment (orucky) [reply doc/60_Dashboard.md]: yes, in the follow-up :-)

**Comment by orucky** [comment]
@32leaves , merging it as all the @JanKoehnlein 's comments were resolved and my further changes are in the other pull requests ...

File: doc/40_Configuration.md (hunk 1 above)
Comment (32leaves) [reply doc/40_Configuration.md]: It is a shell feature, but most prominently used in Docker ... at least in our context. I'm fine with any.

**Pull request merged**